src/flat-button.jsx

```diff
@@ -222,9 +283,9 @@ const FlatButton = React.createClass({
     this.props.onMouseLeave(e);
   },
 
-  _handleTouchStart(e) {
+  _handleTouchTap(e) {
     this.setState({touch: true});
```

Comment:
Regarding this line. It's working like this but it would be better to have the `touch` variable as a instance property rather than an in the state.